charts/nfs-subdir-external-provisioner/README.md

```diff
@@ -69,7 +69,7 @@ The following tables lists the configurable parameters of this chart and their d
 | `leaderElection.enabled`            | Enables or disables leader election                                                                   | `true`                                                   |
 | `nfs.server`                        | Hostname of the NFS server (required)                                                                 | null (ip or hostname)                                    |
 | `nfs.path`                          | Basepath of the mount point to be used                                                                | `/nfs-storage`                                           |
-| `nfs.mountOptions`                  | Mount options (e.g. 'nfsvers=3')                                                                      | null                                                     |
+| `nfs.mountOptions`                  | Mount options (e.g. {"nolock"\\,"nfsvers=3"})                                                          | null                                                     |
```

Comment:
please keep the alignment of `|`

Reply:
also, please rebase your commits to a single commit and `push --force`

Reply:
done.